app/scripts/components/exploration/types.d.ts.ts

```diff
@@ -107,42 +107,54 @@ export interface TimelineDatasetSettings {
 // Tile endpoints for the layer given the current map view.
 type TimelineDatasetMeta = Record<string, any>;
 
-// TimelineDataset type discriminants
-export interface TimelineDatasetIdle {
+// Holds only dataset needed for visualization (Subset of timeline dataset)
+// @ TODO: Rename Timeline specific variable names
+export interface VizDatasetIdle {
   status: TimelineDatasetStatus.IDLE;
   data: EnhancedDatasetLayer;
   error: null;
-  // User controlled settings like visibility, opacity.
   settings: TimelineDatasetSettings;
-  analysis: TimelineDatasetAnalysisIdle;
   meta?: TimelineDatasetMeta;
 }
-export interface TimelineDatasetLoading {
+
+export interface VizDatasetLoading {
   status: TimelineDatasetStatus.LOADING;
   data: EnhancedDatasetLayer;
   error: null;
-  // User controlled settings like visibility, opacity.
   settings: TimelineDatasetSettings;
-  analysis: TimelineDatasetAnalysisIdle;
```

Comment:
The goal is to separate out the attributes necessary only for visualization (for the components that need map visualization), so we can actually use `TimelineDatasets` only for Timeline Datasets. 

I think either works, the current approach or `type VizDatasetIdle = Omit<TimelineDatasetIdle, "analysis">` as you suggested. But in my mind, `VizDataset` is a more fundamental block and `TimelineDataset` is the type derived from it - so `VizDataset + analysis = TimelineDataset` rather than `TimelineDataset - analysis = VizDataset`.